### DISCUSSION TITLE
Are deleted invoices exposed in any APIs ?

### DISCUSSION THREAD

**Discussion by crismas71** [fattureincloud/api#476] [Q&A] [answered]
hello,
in our application we need to know what are the deleted invoices at each connection so that they can be removed in our environment too. Have u got a proper api to retrieve deleted invoices ?

thank u so much

**Comment by valmoz** [comment, marked as answer]
Hello @crismas71 
no, we currently don't have a public API for that.
We suggest instead to use the [Webhooks](https://developers.fattureincloud.it/docs/webhooks/) functionality, using the proper [notification type](https://developers.fattureincloud.it/docs/webhooks/notification-types), to be notified once an invoice is deleted.

Thanks
Mauro

**Comment by crismas71** [comment]
thank u,
I saw the webhook functionality is in beta. when will it be released as part of standard product ?
thks

**Reply by valmoz** [reply]
@crismas71 it should be a matter of some months, we hope by the end of the year.
In the meanwhile, the functionality already seems pretty stable so I would suggest to at least consider it.

Thanks
Mauro
